src/App.tsx

```diff
@@ -1,7 +1,21 @@
+import { ThemeProvider } from "@emotion/react"
+import { theme } from "./theme"
+import {Container, CssBaseline} from "@mui/material"
+import { CustomisedAppBar } from "./components/CustomisedAppBar"
+import FaceScreen from "./components/FaceScreen"
+
 function App() {
 
   return (
-    <></>
+    // <ThemeProvider theme={theme}>
```

Comment:
Why did u comment it?

Reply:
Because I used theme for customising a Button first but then used Styled. And I left it as a cooment in case I use a theme later for other components

Reply:
removed